app/Domain/Projects/Services/Projects.php

```diff
@@ -244,7 +244,8 @@ public function notifyProjectUsers(Notification $notification): void
         {
 
             //Filter notifications
-            $notification = EventCore::dispatchFilter('notificationFilter', $notification);
+            $eventCore = new EventCore();
```

Comment:
Couple things on this one:
1. The event dispatcher is a trait that can be added to a class. 
2. Objects should always be instantiated via the container calls: app()->make(EventCore::class). 
But in this case it should use the Event trait.